sys/linux/dev_kvm.txt

```diff
@@ -232,12 +232,41 @@ kvm_text_x86_64 {
 	size	len[text, intptr]
 }
 
-kvm_text_arm64 {
+kvm_text_arm64 [
+	text_instr	kvm_text_arm64_instr
+	text_api	kvm_text_arm64_syzos_api
+]
+
+kvm_text_arm64_instr {
 	typ	const[0, intptr]
 	text	ptr[in, text[arm64]]
 	size	len[text, intptr]
 }
 
+# Syzos code is organized as a sequence of commands, each occupying 8 int64's.
```

Comment:
I am thinking how we can make it more flexible, e.g. pass strings/binary blobs (instructions for kvm_text_arm64_instr).
We could make each command: {CALL ID, SIZE OF ARGS, ... variable size args ...}. Then the interpreter can pass pointer to args to each api handler, and then skip SIZE OF ARGS bytes.